add-on/jar.mn

```diff
@@ -118,7 +53,16 @@
 % override chrome://loop/skin/toolbar@2x.png      chrome://loop/skin/toolbar-aero@2x.png          os=WINNT osversion=6.1
 % override chrome://loop/skin/toolbar@2x.png      chrome://loop/skin/toolbar-win8@2x.png         os=WINNT osversion=6.2
 % override chrome://loop/skin/toolbar@2x.png      chrome://loop/skin/toolbar-win8@2x.png         os=WINNT osversion=6.3
-  locale/                        (chrome/locale/*)
+  # Due to the way Loop's L10n works, we include a fallback to en-US.
+  content/locale/en-US/          (chrome/locale/en-US/*)
```

Comment:
Yes that's right. I think you're right, its reasonable to move to locale-fallback. Probably will keep the en-US though just for clarity.